apps/build-onchain-apps/contracts/src/CustomERC1155.sol

```diff
@@ -1,25 +1,16 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.13;
 
-import {ERC1155} from '@openzeppelin/contracts/token/ERC1155/ERC1155.sol';
+import {ERC1155} from "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
 
 contract COFFEEKITTEN is ERC1155 {
-  constructor()
-    ERC1155(
-      'https://ipfs.io/ipfs/bafybeiexud2pveyr2ybelpjvwzhpyhw57tefraibf7jn4cecol6qu3jiau/1.json'
-    )
-  {}
+    constructor() ERC1155("https://ipfs.io/ipfs/bafybeiexud2pveyr2ybelpjvwzhpyhw57tefraibf7jn4cecol6qu3jiau/1.json") {}
 
-  function mint(address account, uint256 id, uint256 amount, bytes memory data) public {
-    _mint(account, id, amount, data);
-  }
+    function mint(address account, uint256 id, uint256 amount, bytes memory data) public {
```

Comment:
Is it a thing that Solidity use 4 spaces instead of 2? cc @Sneh1999

Reply:
yes, seems like it: https://docs.soliditylang.org/en/latest/style-guide.html#indentation

Reply:
that's such a Java vibes for 4 spaces. Not sure how to feel about that. eheheh